--- gettext-0.18.1.1/gettext-tools/libgrep/m-regex.c~	2010-06-06 14:49:57.000000000 +0200
+++ gettext-0.18.1.1/gettext-tools/libgrep/m-regex.c	2012-12-12 13:19:17.540818961 +0100
@@ -106,7 +106,7 @@
 
         if ((err = re_compile_pattern (motif, len,
                                        &cregex->patterns[cregex->pcount].regexbuf)) != NULL)
-          error (exit_failure, 0, err);
+          error (exit_failure, 0, "%s", err);
         cregex->pcount++;
 
         motif = sep;
--- gettext-0.18.1.1/gettext-tools/libgrep/m-fgrep.c~	2010-06-06 14:49:57.000000000 +0200
+++ gettext-0.18.1.1/gettext-tools/libgrep/m-fgrep.c	2012-12-12 13:19:46.297484613 +0100
@@ -106,7 +106,7 @@
       for (lim = beg; lim < pattern + pattern_size && *lim != '\n'; ++lim)
         ;
       if ((err = kwsincr (ckwset->kwset, beg, lim - beg)) != NULL)
-        error (exit_failure, 0, err);
+        error (exit_failure, 0, "%s", err);
       if (lim < pattern + pattern_size)
         ++lim;
       beg = lim;
@@ -114,7 +114,7 @@
   while (beg < pattern + pattern_size);
 
   if ((err = kwsprep (ckwset->kwset)) != NULL)
-    error (exit_failure, 0, err);
+    error (exit_failure, 0, "%s", err);
   return ckwset;
 }
 
