--- gettext-0.10.40/configure.in.orig	Wed Jun 12 12:33:34 2002
+++ gettext-0.10.40/configure.in	Wed Jun 12 12:39:13 2002
@@ -43,7 +43,7 @@
 jm_AC_TYPE_UNSIGNED_LONG_LONG
 AC_TYPE_OFF_T
 AC_TYPE_SIZE_T
-AM_TYPE_PTRDIFF_T
+AC_CHECK_TYPES(ptrdiff_t)
 jm_AC_TYPE_UINTMAX_T
 
 dnl Checks for library functions.
@@ -67,7 +67,7 @@
   LIBOBJS="$LIBOBJS printf-prs.o"
 fi
 
-AM_FUNC_ERROR_AT_LINE
+AC_FUNC_ERROR_AT_LINE
 
 gt_SETLOCALE
 
--- gettext-0.10.40/configure.in.orig	Wed Jun 12 12:40:32 2002
+++ gettext-0.10.40/configure.in	Wed Jun 12 12:47:27 2002
@@ -64,7 +64,7 @@
   AC_DEFINE(HAVE_PARSE_PRINTF_FORMAT, 1,
     [Define if you have the parse_printf_format() function.])
 else
-  LIBOBJS="$LIBOBJS printf-prs.o"
+  AC_LIBOBJ([printf-prs])
 fi
 
 AC_FUNC_ERROR_AT_LINE
