diff -Nru gettext-0.10.36/m4/gettext.m4 gettext-0.10.36.new/m4/gettext.m4
--- gettext-0.10.36/m4/gettext.m4	Sat Mar 24 01:51:31 2001
+++ gettext-0.10.36.new/m4/gettext.m4	Wed Apr  4 01:17:27 2001
@@ -159,7 +159,7 @@
       POSUB=po
     fi
     AC_OUTPUT_COMMANDS(
-     [case " $CONFIG_FILES " in *" po/Makefile.in "* | *" po/Makefile.in:"*)
+     [case " $CONFIG_FILES " in *"po/Makefile.in"*)
         sed -e "/POTFILES =/r po/POTFILES" po/Makefile.in > po/Makefile
       esac])
 
