--- gettext-0.20.2/gettext-runtime/gnulib-m4/lib-prefix.m4.orig	2020-04-21 00:29:38.614580831 +0200
+++ gettext-0.20.2/gettext-runtime/gnulib-m4/lib-prefix.m4	2020-04-21 00:42:56.945602835 +0200
@@ -283,6 +283,7 @@
                case "$searchdir" in
                  */lib32/ | */lib32 ) acl_libdirstem2=lib32 ;;
                  */lib64/ | */lib64 ) acl_libdirstem3=lib64 ;;
+                 */libx32/ | */libx32 ) acl_libdirstem3=libx32 ;;
                  */../ | */.. )
                    # Better ignore directories of this form. They are misleading.
                    ;;
@@ -290,6 +291,7 @@
                     case "$searchdir" in
                       */lib32 ) acl_libdirstem2=lib32 ;;
                       */lib64 ) acl_libdirstem3=lib64 ;;
+                      */libx32 ) acl_libdirstem3=libx32 ;;
                     esac ;;
                esac
              fi
--- gettext-0.19.8.1/gettext-runtime/gnulib-m4/lib-link.m4.orig	2016-06-11 14:59:32.000000000 +0200
+++ gettext-0.19.8.1/gettext-runtime/gnulib-m4/lib-link.m4	2017-02-05 14:27:31.840415976 +0100
@@ -370,7 +370,12 @@
           fi
           if test "X$found_dir" != "X"; then
             dnl Found the library.
+
+            if test "X$found_dir" = "X/usr/$acl_libdirstem" || test "X$found_dir" = "X/usr/$acl_libdirstem2"; then
+            LTLIB[]NAME="${LTLIB[]NAME}${LTLIB[]NAME:+ }-l$name"
+            else
             LTLIB[]NAME="${LTLIB[]NAME}${LTLIB[]NAME:+ }-L$found_dir -l$name"
+            fi
             if test "X$found_so" != "X"; then
               dnl Linking with a shared library. We attempt to hardcode its
               dnl directory into the executable's runpath, unless it's the
--- gettext-0.20.2/gettext-tools/gnulib-m4/lib-prefix.m4.orig	2020-04-21 00:29:38.614580831 +0200
+++ gettext-0.20.2/gettext-tools/gnulib-m4/lib-prefix.m4	2020-04-21 00:45:17.754663042 +0200
@@ -283,6 +283,7 @@
                case "$searchdir" in
                  */lib32/ | */lib32 ) acl_libdirstem2=lib32 ;;
                  */lib64/ | */lib64 ) acl_libdirstem3=lib64 ;;
+                 */libx32/ | */libx32 ) acl_libdirstem3=libx32 ;;
                  */../ | */.. )
                    # Better ignore directories of this form. They are misleading.
                    ;;
@@ -290,6 +291,7 @@
                     case "$searchdir" in
                       */lib32 ) acl_libdirstem2=lib32 ;;
                       */lib64 ) acl_libdirstem3=lib64 ;;
+                      */libx32 ) acl_libdirstem3=libx32 ;;
                     esac ;;
                esac
              fi
--- gettext-0.19.8.1/gettext-tools/gnulib-m4/lib-link.m4.orig	2016-06-11 14:59:32.000000000 +0200
+++ gettext-0.19.8.1/gettext-tools/gnulib-m4/lib-link.m4	2017-02-05 14:27:31.840415976 +0100
@@ -370,7 +370,12 @@
           fi
           if test "X$found_dir" != "X"; then
             dnl Found the library.
+
+            if test "X$found_dir" = "X/usr/$acl_libdirstem" || test "X$found_dir" = "X/usr/$acl_libdirstem2"; then
+            LTLIB[]NAME="${LTLIB[]NAME}${LTLIB[]NAME:+ }-l$name"
+            else
             LTLIB[]NAME="${LTLIB[]NAME}${LTLIB[]NAME:+ }-L$found_dir -l$name"
+            fi
             if test "X$found_so" != "X"; then
               dnl Linking with a shared library. We attempt to hardcode its
               dnl directory into the executable's runpath, unless it's the
