--- gettext-0.11.2/configure.in.orig	Thu May 16 19:59:15 2002
+++ gettext-0.11.2/configure.in	Thu May 16 20:02:24 2002
@@ -40,6 +40,7 @@
 
 dnl Check for build configuration.
 AM_PROG_LIBTOOL
+AM_PROG_LEX
 
 AC_DEFINE(_GNU_SOURCE, 1, [Make sure we see all GNU extensions.])
 
@@ -58,7 +58,7 @@
 jm_AC_TYPE_UNSIGNED_LONG_LONG
 AC_TYPE_OFF_T
 AC_TYPE_SIZE_T
-AM_TYPE_PTRDIFF_T
+AC_CHECK_TYPES(ptrdiff_t)
 jm_AC_TYPE_UINTMAX_T
 gt_TYPE_SSIZE_T
 AC_TYPE_PID_T
@@ -70,7 +70,6 @@
 strerror strtoul uname utime utimes])
 AC_REPLACE_FUNCS([memset stpcpy strcspn \
 strcasecmp strncasecmp strpbrk strstr vasprintf])
-AM_FUNC_GETLINE
 if test $am_cv_func_working_getline != yes; then
   AC_CHECK_FUNCS(getdelim)
 fi
@@ -135,7 +135,7 @@
   AC_MSG_CHECKING([whether the C++ compiler ($CXX $CXXFLAGS $LDFLAGS) works])
   AC_LANG_SAVE
   AC_LANG_CPLUSPLUS
-  AC_TRY_COMPILER([int main(){return(0);}], ac_cv_prog_cxx_works, ac_cv_prog_cxx_cross)
+  AC_TRY_COMPILE([],[int main(){return(0);}], ac_cv_prog_cxx_works, ac_cv_prog_cxx_cross)
   AC_LANG_RESTORE
   AC_MSG_RESULT($ac_cv_prog_cxx_works)
   if test $ac_cv_prog_cxx_works = no; then
@@ -158,17 +158,6 @@
 
 EOF
 
-dnl These can be removed when we drop support for autoconf-2.13 and switch
-dnl to autoconf-2.52.
-changequote(,)dnl
-LTLIBOBJS=`echo "$LIBOBJS" | sed 's/\.[^.]* /.lo /g;s/\.[^.]*$/.lo/'`
-changequote([, ])dnl
-AC_SUBST(LTLIBOBJS)
-changequote(,)dnl
-LTALLOCA=`echo "$ALLOCA" | sed 's/\.[^.]* /.lo /g;s/\.[^.]*$/.lo/'`
-changequote([, ])dnl
-AC_SUBST(LTALLOCA)
-
 AC_OUTPUT([Makefile \
            lib/Makefile lib/javacomp.sh lib/javaexec.sh \
            intl/Makefile intl-java/Makefile \
--- gettext-0.11.2/m4/lib-link.m4.orig	Tue Apr  9 15:07:04 2002
+++ gettext-0.11.2/m4/lib-link.m4	Thu May 16 20:29:17 2002
@@ -64,7 +64,7 @@
   ac_save_CPPFLAGS="$CPPFLAGS"
   AC_LIB_APPENDTOVAR([CPPFLAGS], [$INC]NAME)
 
-  AC_CACHE_CHECK([for lib[]$1], [ac_cv_lib[]Name], [
+  AC_CACHE_CHECK([for lib[]$1], ac_cv_lib[]Name, [
     ac_save_LIBS="$LIBS"
     LIBS="$LIBS $LIB[]NAME"
     AC_TRY_LINK([$3], [$4], [ac_cv_lib[]Name=yes], [ac_cv_lib[]Name=no])
--- gettext-0.11.2/m4/fnmatch.m4.orig	Mon Feb  4 14:32:49 2002
+++ gettext-0.11.2/m4/fnmatch.m4	Thu May 16 20:30:48 2002
@@ -44,7 +44,6 @@
     dnl libgettextlib.so.
     AC_DEFINE([fnmatch], [posix_fnmatch],
       [Define to a replacement function name for fnmatch().])
-    LIBOBJS="$LIBOBJS pfnmatch.${ac_objext}"
-    AC_SUBST(LIBOBJS)
+    AC_LIBOBJ([pfnmatch])
   fi
 ])
--- gettext-0.11.2/m4/getline.m4.orig	Wed Jan 16 13:19:34 2002
+++ gettext-0.11.2/m4/getline.m4	Thu May 16 20:31:00 2002
@@ -47,7 +47,6 @@
     dnl libgettextlib.so.
     AC_DEFINE([getline], [gnu_getline],
       [Define to a replacement function name for getline().])
-    LIBOBJS="$LIBOBJS getline.${ac_objext}"
-    AC_SUBST(LIBOBJS)
+    AC_LIBOBJ([getline])
   fi
 ])
--- gettext-0.11.2/lib/Makefile.am.orig	Mon Mar 11 13:19:23 2002
+++ gettext-0.11.2/lib/Makefile.am	Thu May 16 20:37:13 2002
@@ -87,8 +87,6 @@
 
 # How to build libgettextlib.la.
 
-libgettextlib_la_LIBADD = @LTALLOCA@ @LTLIBOBJS@
-
 # Need @LTLIBINTL@ because many source files use gettext().
 # Need @LTLIBICONV@ because linebreak.c uses iconv().
 libgettextlib_la_LDFLAGS = \
