diff -Nru gettext-0.12.orig/gettext-tools/doc/Makefile.am gettext-0.12/gettext-tools/doc/Makefile.am
--- gettext-0.12.orig/gettext-tools/doc/Makefile.am	2003-04-14 15:39:49.000000000 +0200
+++ gettext-0.12/gettext-tools/doc/Makefile.am	2003-05-20 14:51:18.000000000 +0200
@@ -17,19 +17,10 @@
 
 ## Process this file with automake to produce Makefile.in.
 
-docdir = $(datadir)/doc/gettext
-dvidir = $(docdir)
-psdir = $(docdir)
-pdfdir = $(docdir)
-htmldir = $(docdir)
-
 AUTOMAKE_OPTIONS = 1.2 gnits
 EXTRA_DIST =
 MOSTLYCLEANFILES =
 
-SED = sed
-RM = rm -f
-
 MAKEINFO = env LANG= LANGUAGE= @MAKEINFO@
 
 info_TEXINFOS = gettext.texi
@@ -55,150 +46,6 @@
 	mv iso-3166.tmp $(srcdir)/iso-3166.texi
 
 
-# We distribute only the split HTML documentation.
-# The user can generate the others, via
-#   make gettext.ps
-#   make gettext.pdf
-#   make gettext.html
-
-all-local: html
-install-data-local: install-html
-installdirs-local: installdirs-html
-uninstall-local: uninstall-html
-# Hidden from automake, but really activated. Works around an automake-1.5 bug.
-#distdir: dist-html
-
-html: html-split
-install-html: install-html-split
-uninstall-html: uninstall-html-split
-dist-html: dist-html-split
-
-CLEANFILES = gettext.pdf gettext.html
-# gettext.dvi and gettext.ps are already known to automake.
-MAINTAINERCLEANFILES = gettext_*.html
-
-
-# Documentation in DVI format.
-
-install-dvi: gettext.dvi
-	$(mkinstalldirs) $(DESTDIR)$(dvidir)
-	$(INSTALL_DATA) `if test -f gettext.dvi; then echo .; else echo $(srcdir); fi`/gettext.dvi $(DESTDIR)$(dvidir)/gettext.dvi
-
-installdirs-dvi:
-	$(mkinstalldirs) $(DESTDIR)$(dvidir)
-
-uninstall-dvi:
-	$(RM) $(DESTDIR)$(dvidir)/gettext.dvi
-
-# Temporary index files. automake removes only the predefined ones by itself.
-MOSTLYCLEANFILES += \
-  gettext.am gettext.ams \
-  gettext.cp gettext.cps \
-  gettext.ef gettext.efs \
-  gettext.em gettext.ems \
-  gettext.ev gettext.evs \
-  gettext.fn gettext.fns \
-  gettext.kw gettext.kws \
-  gettext.op gettext.ops \
-  gettext.pg gettext.pgs \
-  gettext.vr gettext.vrs
-
-
-# Documentation in Postscript format.
-
-DVIPS = @DVIPS@ -D600
-
-ps: gettext.ps
-
-gettext.ps: gettext.dvi
-	$(DVIPS) -o $@ `if test -f gettext.dvi; then echo gettext.dvi; else echo $(srcdir)/gettext.dvi; fi`
-
-install-ps: gettext.ps
-	$(mkinstalldirs) $(DESTDIR)$(psdir)
-	$(INSTALL_DATA) `if test -f gettext.ps; then echo .; else echo $(srcdir); fi`/gettext.ps $(DESTDIR)$(psdir)/gettext.ps
-
-installdirs-ps:
-	$(mkinstalldirs) $(DESTDIR)$(psdir)
-
-uninstall-ps:
-	$(RM) $(DESTDIR)$(psdir)/gettext.ps
-
-
-# Documentation in Portable Document Format.
-
-TEXI2PDF = @TEXI2PDF@
-SUFFIXES = .pdf
-
-pdf: gettext.pdf
-
-.texi.pdf:
-	TEXINPUTS=.:$$TEXINPUTS \
-	  MAKEINFO='$(MAKEINFO) -I $(srcdir)' $(TEXI2PDF) $<
-
-install-pdf: gettext.pdf
-	$(mkinstalldirs) $(DESTDIR)$(pdfdir)
-	$(INSTALL_DATA) `if test -f gettext.pdf; then echo .; else echo $(srcdir); fi`/gettext.pdf $(DESTDIR)$(pdfdir)/gettext.pdf
-
-installdirs-pdf:
-	$(mkinstalldirs) $(DESTDIR)$(pdfdir)
-
-uninstall-pdf:
-	$(RM) $(DESTDIR)$(pdfdir)/gettext.pdf
-
-
-# Documentation in HTML format.
-
-TEXI2HTML = @PERL@ $(srcdir)/texi2html
-
-html-monolithic: gettext.html
-html-split: gettext_toc.html
-
-gettext.html: gettext.texi version.texi $(gettext_TEXINFOS)
-	$(TEXI2HTML) -expandinfo -number -monolithic `if test -f gettext.texi; then echo gettext.texi; else echo $(srcdir)/gettext.texi; fi`
-
-gettext_toc.html: gettext.texi version.texi $(gettext_TEXINFOS)
-	case "@PERL@" in \
-	  *"/missing perl") \
-	     $(TEXI2HTML) -expandinfo -number -split_chapter `if test -f gettext.texi; then echo gettext.texi; else echo $(srcdir)/gettext.texi; fi` || exit 0 ;; \
-	  *) $(RM) gettext_*.html ; \
-	     $(TEXI2HTML) -expandinfo -number -split_chapter `if test -f gettext.texi; then echo gettext.texi; else echo $(srcdir)/gettext.texi; fi` ;; \
-	esac
-
-install-html-monolithic: gettext.html
-	$(mkinstalldirs) $(DESTDIR)$(htmldir)
-	$(INSTALL_DATA) `if test -f gettext.html; then echo .; else echo $(srcdir); fi`/gettext.html $(DESTDIR)$(htmldir)/gettext.html
-
-install-html-split: gettext_toc.html
-	$(mkinstalldirs) $(DESTDIR)$(htmldir)
-	for file in `if test -f gettext_toc.html; then echo .; else echo $(srcdir); fi`/gettext_*.html; do \
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(htmldir)/`basename $$file`; \
-	done
-
-installdirs-html:
-	$(mkinstalldirs) $(DESTDIR)$(htmldir)
-
-uninstall-html-monolithic:
-	$(RM) $(DESTDIR)$(htmldir)/gettext.html
-
-uninstall-html-split:
-	$(RM) $(DESTDIR)$(htmldir)/gettext_*.html
-
-dist-html-monolithic:
-	$(mkinstalldirs) $(distdir)/
-	file=gettext.html; \
-	if test -f $$file; then d=.; else d=$(srcdir); fi; \
-	cp -p $$d/$$file $(distdir)/$$file || exit 1
-
-# We would like to put gettext_*.html into EXTRA_DIST, but it doesn't work.
-dist-html-split:
-	$(mkinstalldirs) $(distdir)/
-	file=gettext_toc.html; \
-	if test -f $$file; then d=.; else d=$(srcdir); fi; \
-	for file in `cd $$d && echo gettext_*.html`; do \
-	  cp -p $$d/$$file $(distdir)/$$file || exit 1; \
-	done
-
-
 # VMS support.
 
 EXTRA_DIST += Makefile.vms
