diff -urN gettext-0.10.37.org/m4/gettext.m4 gettext-0.10.37/m4/gettext.m4
--- gettext-0.10.37.org/m4/gettext.m4	Fri Apr 20 10:06:56 2001
+++ gettext-0.10.37/m4/gettext.m4	Fri Apr 20 10:07:14 2001
@@ -159,7 +159,7 @@
       POSUB=po
     fi
     AC_OUTPUT_COMMANDS(
-     [case " "$CONFIG_FILES" " in *" po/Makefile.in "* | *" po/Makefile.in:"*)
+     [case " "$CONFIG_FILES" " in *"po/Makefile.in"*)
         sed -e "/POTFILES =/r po/POTFILES" po/Makefile.in > po/Makefile
         ;;
       esac])
