--- gettext-0.11.5/misc/gettextize.in.orig	Sun Nov 24 22:26:59 2002
+++ gettext-0.11.5/misc/gettextize.in	Sun Nov 24 22:27:43 2002
@@ -869,7 +869,7 @@
 func_modify_configure_in "(AC_OUTPUT): Remove command that created po/Makefile."
 sed -e '/^\(dnl \|\)AC_LINK_FILES(\$nls_cv_header_libgt, \$nls_cv_header_intl)$/d' < "$srcdir/$configure_in" > "$srcdir/$configure_in.tmp"
 func_modify_configure_in "(AC_LINK_FILES): Remove invocation."
-sed -e 's/^AM_GNU_GETTEXT_VERSION([^()]*)/AM_GNU_GETTEXT_VERSION('"$version"'/' < "$srcdir/$configure_in" > "$srcdir/$configure_in.tmp"
+sed -e 's/^AM_GNU_GETTEXT_VERSION([^()]*)/AM_GNU_GETTEXT_VERSION('"$version"')/' < "$srcdir/$configure_in" > "$srcdir/$configure_in.tmp"
 func_modify_configure_in "(AM_GNU_GETTEXT_VERSION): Bump to $version."
 $do_changelog && func_ChangeLog_finish
 
