diff -Nru gettext-0.12.orig/gettext-tools/configure.ac gettext-0.12/gettext-tools/configure.ac
--- gettext-0.12.orig/gettext-tools/configure.ac	2003-05-12 21:47:48.000000000 +0200
+++ gettext-0.12/gettext-tools/configure.ac	2003-05-20 15:24:37.000000000 +0200
@@ -230,10 +230,7 @@
 
 AC_CONFIG_FILES([Makefile])
 
-AC_CONFIG_FILES([doc/Makefile],
-                [FIX_MAKEFILE_COMPILE([doc/Makefile])
-                 FIX_MAKEFILE_DISTRIB([doc/Makefile])
-                 FIX_MAKEFILE_INFO([doc/Makefile])])
+AC_CONFIG_FILES([doc/Makefile])
 
 AC_CONFIG_FILES([intl/Makefile:../gettext-runtime/intl/Makefile.in], [
   # Change srcdir variable so that it points to ../gettext-runtime/intl.
@@ -242,15 +239,12 @@
   mv intl/Makefile.tmp intl/Makefile
   ])
 
-AC_CONFIG_FILES([lib/Makefile],
-                [FIX_MAKEFILE_COMPILE([lib/Makefile])])
+AC_CONFIG_FILES([lib/Makefile])
 AC_CONFIG_FILES([lib/javacomp.sh lib/javaexec.sh])
 
-AC_CONFIG_FILES([libuniname/Makefile],
-                [FIX_MAKEFILE_COMPILE([libuniname/Makefile])])
+AC_CONFIG_FILES([libuniname/Makefile])
 
-AC_CONFIG_FILES([src/Makefile],
-                [FIX_MAKEFILE_COMPILE([src/Makefile])])
+AC_CONFIG_FILES([src/Makefile])
 AC_CONFIG_FILES([src/user-email])
 
 AC_CONFIG_FILES([po/Makefile.in])
@@ -258,15 +252,14 @@
 AC_CONFIG_FILES([projects/Makefile])
 
 AC_CONFIG_FILES([misc/Makefile])
-AC_CONFIG_FILES([misc/gettextize], [chmod a+x misc/gettextize])
-AC_CONFIG_FILES([misc/autopoint], [chmod a+x misc/autopoint])
+AC_CONFIG_FILES([misc/gettextize])
+AC_CONFIG_FILES([misc/autopoint])
 
 AC_CONFIG_FILES([man/Makefile])
 AC_CONFIG_FILES([man/x-to-1])
 
 AC_CONFIG_FILES([m4/Makefile])
 
-AC_CONFIG_FILES([tests/Makefile],
-                [FIX_MAKEFILE_COMPILE([tests/Makefile])])
+AC_CONFIG_FILES([tests/Makefile])
 
 AC_OUTPUT
