--- gettext-0.11.4/misc/gettextize.in.orig	Tue Jul 23 19:35:14 2002
+++ gettext-0.11.4/misc/gettextize.in	Wed Jul 31 19:49:46 2002
@@ -965,10 +965,6 @@
     5) count="five paragraphs";;
     *) count="$count paragraphs";;
   esac
-  echo "Press Return to acknowledge the previous $count."
-  # Read from /dev/tty, not stdin, so that gettextize cannot be abused by
-  # non-interactive tools.
-  read < /dev/tty
 fi
 
 exit 0
