--- gettext-0.10.38/m4/gettext.m4.wiget	Thu May 17 15:35:37 2001
+++ gettext-0.10.38/m4/gettext.m4	Fri Jun 29 16:07:08 2001
@@ -181,15 +181,15 @@
           ac_dir=`echo "$ac_file"|sed 's%/[^/][^/]*$%%'`
           ac_dir_suffix="/`echo "$ac_dir"|sed 's%^\./%%'`"
           ac_dots=`echo "$ac_dir_suffix"|sed 's%/[^/]*%../%g'`
-          case "$ac_given_srcdir" in
+          case "$srcdir" in
             .)  top_srcdir=`echo $ac_dots|sed 's%/$%%'` ;;
-            /*) top_srcdir="$ac_given_srcdir" ;;
-            *)  top_srcdir="$ac_dots$ac_given_srcdir" ;;
+            /*) top_srcdir="$srcdir" ;;
+            *)  top_srcdir="$ac_dots$srcdir" ;;
           esac
-          if test -f "$ac_given_srcdir/$ac_dir/POTFILES.in"; then
+          if test -f "$srcdir/$ac_dir/POTFILES.in"; then
             rm -f "$ac_dir/POTFILES"
             echo creating "$ac_dir/POTFILES"
-            sed -e "/^#/d" -e "/^[ 	]*\$/d" -e "s,.*,     $top_srcdir/& \\\\," -e "\$s/\(.*\) \\\\/\1/" < "$ac_given_srcdir/$ac_dir/POTFILES.in" > "$ac_dir/POTFILES"
+            sed -e "/^#/d" -e "/^[ 	]*\$/d" -e "s,.*,     $top_srcdir/& \\\\," -e "\$s/\(.*\) \\\\/\1/" < "$srcdir/$ac_dir/POTFILES.in" > "$ac_dir/POTFILES"
             echo creating "$ac_dir/Makefile"
             sed -e "/POTFILES =/r $ac_dir/POTFILES" "$ac_dir/Makefile.in" > "$ac_dir/Makefile"
           fi
