diff -ur gettext-0.17.old/gettext-runtime/libasprintf/configure.ac gettext-0.17/gettext-runtime/libasprintf/configure.ac
--- gettext-0.17.old/gettext-runtime/libasprintf/configure.ac	2007-11-10 19:00:24.000000000 +0100
+++ gettext-0.17/gettext-runtime/libasprintf/configure.ac	2007-11-10 19:01:22.000000000 +0100
@@ -54,8 +54,7 @@
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_INLINE
 AC_TYPE_SIZE_T
-gl_AC_TYPE_LONG_LONG
-gt_TYPE_LONGDOUBLE
+AC_TYPE_LONG_LONG_INT
 gt_TYPE_WCHAR_T
 gt_TYPE_WINT_T
 AC_CHECK_TYPE([ptrdiff_t], ,
Only in gettext-0.17/gettext-runtime/libasprintf: configure.ac~
