--- gettext-0.13.1/gettext-tools/lib/Makefile.am.orig	2003-11-16 16:20:38.000000000 +0100
+++ gettext-0.13.1/gettext-tools/lib/Makefile.am	2003-12-19 00:13:44.377267408 +0100
@@ -116,7 +116,7 @@
   @LTLIBINTL@ @LTLIBICONV@ -lc -no-undefined
 
 # No need to install libgettextlib.a.
-install-exec-am: install-exec-clean
+install-exec-hook: install-exec-clean
 install-exec-clean:
 	$(RM) $(DESTDIR)$(libdir)/libgettextlib.a
 
--- gettext-0.13.1/gettext-tools/src/Makefile.am.orig	2003-11-16 16:20:38.000000000 +0100
+++ gettext-0.13.1/gettext-tools/src/Makefile.am	2003-12-19 00:13:27.739796688 +0100
@@ -146,7 +146,7 @@
   ../lib/libgettextlib.la @LTLIBINTL@ @LTLIBICONV@ -lc -no-undefined
 
 # No need to install libgettextsrc.a.
-install-exec-am: install-exec-clean
+install-exec-hook: install-exec-clean
 install-exec-clean:
 	$(RM) $(DESTDIR)$(libdir)/libgettextsrc.a
 
