diff -urN gettext-0.17.org/gettext-tools/misc/autopoint.in gettext-0.17/gettext-tools/misc/autopoint.in
--- gettext-0.17.org/gettext-tools/misc/autopoint.in	2007-11-07 03:53:58.000000000 +0100
+++ gettext-0.17/gettext-tools/misc/autopoint.in	2009-08-19 14:10:49.914980544 +0200
@@ -339,8 +339,6 @@
 # a severe restrictions, because most of the people who use autopoint are
 # users of CVS.
 #
-# Check availability of the CVS program.
-(cvs -v) >/dev/null 2>/dev/null || func_fatal_error "cvs program not found"
 
 # Check in which directory config.rpath, mkinstalldirs etc. belong.
 auxdir=`cat "$configure_in" | grep '^AC_CONFIG_AUX_DIR' | sed -n -e 's/AC_CONFIG_AUX_DIR(\([^()]*\))/\1/p' | sed -e 's/^\[\(.*\)\]$/\1/' | sed -e 1q`
@@ -381,17 +379,7 @@
 # Set variables
 # - cvs_dir         directory containing the temporary repository
 # - work_dir        directory containing the temporary checkout
-cvs_dir=tmpcvs$$
 work_dir=tmpwrk$$
-# Use an umask of 077, to avoid attacks that work by overwriting files in the
-# "$CVSROOT"/CVSROOT directory.
-(umask 077 && mkdir "$cvs_dir") || {
-  if test -d "$cvs_dir"; then
-    func_fatal_error "directory $cvs_dir already exists"
-  else
-    func_fatal_error "cannot create directory $cvs_dir"
-  fi
-}
 mkdir "$work_dir" || {
   if test -d "$work_dir"; then
     func_fatal_error "directory $work_dir already exists"
@@ -399,34 +387,13 @@
     func_fatal_error "cannot create directory $work_dir"
   fi
 }
-CVSROOT="$srcdir/$cvs_dir"
-export CVSROOT
-unset CVS_CLIENT_LOG
-unset CVS_CLIENT_PORT
-unset CVS_IGNORE_REMOTE_ROOT
-unset CVS_PASSFILE
-unset CVS_PASSWORD
-unset CVS_RCMD_PORT
-unset CVS_RSH
-unset CVS_SERVER
-unset CVS_SERVER_SLEEP
-unset CVSIGNORE
-unset CVSREAD
-unset CVSUMASK
-unset CVSWRAPPERS
-
-# Need to pass -d "$CVSROOT", because there may be a CVS directory in the
-# current directory.
-cvs -d "$CVSROOT" init
-gzip -d -c < "$gettext_dir/archive.tar.gz" | (cd "$cvs_dir" && tar xf -)
 
 cd "$work_dir"
 cvsver=gettext-`echo "$ver" | sed -e 's/\./_/g'`
-(cvs checkout -r"$cvsver" archive > /dev/null) 2>&1 | grep -v '^cvs checkout: Updating'
-find archive -name CVS -type d -print | xargs rm -rf
-if test `find archive -type f -print | wc -l` = 0; then
-  cd ..
-  rm -rf "$cvs_dir" "$work_dir"
+if test -f "$gettext_dir/archive-$cvsver.tar.gz"; then
+	tar xzf "$gettext_dir/archive-$cvsver.tar.gz"
+else
+  rm -rf "$work_dir"
   func_fatal_error "infrastructure files for version $ver not found; this is autopoint from GNU $package $version"
 fi
 cd ..
@@ -489,7 +456,7 @@
     fi
   done
   if test -n "$mismatch"; then
-    rm -rf "$cvs_dir" "$work_dir"
+    rm -rf "$work_dir"
     func_fatal_error "Some files have been locally modified. Not overwriting them because --force has not been specified. For your convenience, you find the local modifications in the file '$mismatchfile'."
   fi
   rm -rf "$tmp"
